Makefile

```diff
@@ -137,7 +137,7 @@ examples/%.ttl: src/data/examples/%.yaml
 
 test-examples: examples/output
 
-examples/output: src/mixs_6_2_for_merge/schema/mixs_6_2_for_merge.yaml
+examples/output: src/mixs
```

Comment:
Is `src/mixs` the right target that we want the `examples/output` target to depend on?